TodoListClient/MainWindow.xaml.cs

```diff
@@ -245,7 +250,7 @@ private async void SignIn(object sender = null, RoutedEventArgs args = null)
             if (SignInButton.Content.ToString() == "Clear Cache")
             {
                 TodoList.ItemsSource = string.Empty;
-                ClearCache(app);
+                await ClearCache(app);
```

Comment:
Github wouldn't allow me to add comments on lines that were not modified, but on line 268, I believe you should add an await too:

await GetTodoList();

@jmprieur

Reply:
thanks @andretms. actually I missed it on purpose: we don't want OnInitialized() to be blocking, otherwise the UI gets frozen